Xamarin.Forms.Platform.Android/PopupManager.cs

```diff
@@ -112,9 +112,58 @@ void OnActionSheetRequested(Page sender, ActionSheetArguments arguments)
 				builder.Dispose();
 				//to match current functionality of renderer we set cancelable on outside
 				//and return null
+				if (arguments.FlowDirection == FlowDirection.MatchParent)
```

Comment:
I think this whole if statement and the `SetTitleFlowDirection` method can just replaced by doing this

```C#
if (arguments.FlowDirection == FlowDirection.MatchParent && sender is IVisualElementController ve)
{
     dialog.Window.DecorView.UpdateFlowDirection(ve);
}
```